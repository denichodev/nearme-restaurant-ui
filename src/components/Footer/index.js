import React from 'react';
import { Link } from 'react-router-dom';

import { Grid, Row, Col } from '../Layout';
import s from './styles.css';

const Footer = () => {
  return (
    <footer className={s.container}>
      <Grid className={s.desktop}>
        <Row>
          <Col xs={12}>
            <div className={s.wrapper}>
              <div className={s.copyright}>
                &copy; Aviato.co 2018
              </div>
              <div className={s.menu}>
                <ul>
                  <li><Link to="">About</Link></li>
                  <li><Link to="">Terms</Link></li>
                  <li><Link to="">Privacy Policy</Link></li>
                </ul>
              </div>
            </div>
          </Col>
        </Row>
      </Grid>
      <Grid className={s.mobile}>
        <Row>
          <Col xs={12}>
            <div className={s.mobileNav}>
              <div>
                <Link to="/">
                  <i className="fas fa-home" />
                </Link>
              </div>
              <div>
                <Link to="/explore">
                  <i className="fas fa-search" />
                </Link>
              </div>
              <div>
                <Link to="/reserved">
                  <i className="fas fa-heart" />
                </Link>
              </div>
              <div>
                <Link to="/me">
                  <i className="fas fa-user" />
                </Link>
              </div>
            </div>
          </Col>
        </Row>
      </Grid>
    </footer>
  );
}

export default Footer;
