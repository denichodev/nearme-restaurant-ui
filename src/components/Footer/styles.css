.container {
  margin-top: 50px;
}

@media (max-width: 600px) {
  :global #root {
    padding-bottom: 10px !important;
  }

  .mobile {
    display: block;
    position: fixed;
    bottom: 0;
    width: 100%;
    background-color: #FFF;
    border-top: 1px solid #DBDBDB;
    z-index: 99;
  }

  .mobileNav {
    display: flex;
    width: 100%;
    flex-flow: row wrap;
    justify-content: space-between;
  }

  .mobileNav > div {
    text-align: center;
    width: 25%;
    line-height: 40px;
  }

  .mobileNav button i {
    font-size: 24px;
  }

  .mobileNav a {
    display: block;
  }

  .desktop {
    display: none;
  }
}

@media (min-width: 600px) {
  .mobile {
    display: none;
  }

  .desktop {
    display: block;
  }

  .wrapper {
    display: flex;
    border-top: 1px solid #DBDBDB;
    justify-content: space-between;
  }

  .copyright {
    font-size: 14px;
    padding: 20px 0;
  }

  .menu {

  }

  .menu ul {
    list-style: none;
    margin: 0;
    padding: 0;
  }

  .menu ul li {
    display: inline-block;
  }

  .menu ul li a {
    display: inline-block;
    padding: 20px 10px;
    font-size: 14px;
  }
}