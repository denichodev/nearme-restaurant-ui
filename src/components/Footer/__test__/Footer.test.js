import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Footer from '../index';

describe('<Footer />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Footer />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
