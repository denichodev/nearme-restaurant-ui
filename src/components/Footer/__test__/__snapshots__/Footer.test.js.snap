// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<footer
  className="container"
>
  <Grid
    className="desktop"
    fluid={false}
  >
    <Row>
      <Col
        xs={12}
      >
        <div
          className="wrapper"
        >
          <div
            className="copyright"
          >
            Â© Aviato.co 2018
          </div>
          <div
            className="menu"
          >
            <ul>
              <li>
                <Link
                  replace={false}
                  to=""
                >
                  About
                </Link>
              </li>
              <li>
                <Link
                  replace={false}
                  to=""
                >
                  Terms
                </Link>
              </li>
              <li>
                <Link
                  replace={false}
                  to=""
                >
                  Privacy Policy
                </Link>
              </li>
            </ul>
          </div>
        </div>
      </Col>
    </Row>
  </Grid>
  <Grid
    className="mobile"
    fluid={false}
  >
    <Row>
      <Col
        xs={12}
      >
        <div
          className="mobileNav"
        >
          <div>
            <Link
              replace={false}
              to="/"
            >
              <i
                className="fas fa-home"
              />
            </Link>
          </div>
          <div>
            <Link
              replace={false}
              to="/explore"
            >
              <i
                className="fas fa-search"
              />
            </Link>
          </div>
          <div>
            <Link
              replace={false}
              to="/reserved"
            >
              <i
                className="fas fa-heart"
              />
            </Link>
          </div>
          <div>
            <Link
              replace={false}
              to="/me"
            >
              <i
                className="fas fa-user"
              />
            </Link>
          </div>
        </div>
      </Col>
    </Row>
  </Grid>
</footer>
`;
